### ISSUE TITLE
overfitting problem

### ISSUE THREAD

**Issue by Rimehdaoudi** [matterport/Mask_RCNN#2382]
hi am trying this implementation on my own dataset and while visualize the loss function curve a realize that there's an overfitting problem. also when i get a different mAP every time i run the model on the test set images( the test images are used only for test)! can anyone tell me what happen.
here's the loss curve

![image](https://user-images.githubusercontent.com/45721953/95002948-e00e2e80-05d9-11eb-9369-9b03d51e3d70.png)

**Comment by pafechet** [comment]
Hi @Rimehdaoudi
How much images you have for training and validation ?
How much class and are they well distributed ?

**Comment by Rimehdaoudi** [comment]
Hi @pafechet 
A total of 540 images were selected randomly and used
for the model training with 80% for training and 20% for
validation
for the classes it's for brain tumor detection so we have tumor + background

**Comment by pafechet** [comment]
@Rimehdaoudi 
Ok you have quite a good amount of image for mask r-cnn, actually it's a transfert learning model so you could use even less. 
Well im sorry i don't really know right now. 
Check your bacth size too....

**Comment by konstantin-frolov** [comment]
You don't have overfitting problem. Overfitting is when val loss start increase after decreasing, but train loss сontinues to decrease.
